From 0247d97ca455b3d583a80a8b3b1a67cce2d44b5b Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 28 Dec 2022 01:01:35 +0800
Subject: [PATCH 05/28] Revert "drm/display/dp_mst: Fix
 drm_dp_mst_add_affected_dsc_crtcs() return code"

This reverts commit 44b204730bf32131f064ebc90145092590c7fe95.
---
 drivers/gpu/drm/drm_dp_mst_topology.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_dp_mst_topology.c b/drivers/gpu/drm/drm_dp_mst_topology.c
index 0feeac52e4eb3..4272cd3622f8b 100644
--- a/drivers/gpu/drm/drm_dp_mst_topology.c
+++ b/drivers/gpu/drm/drm_dp_mst_topology.c
@@ -5238,7 +5238,7 @@ int drm_dp_mst_add_affected_dsc_crtcs(struct drm_atomic_state *state, struct drm
 	mst_state = drm_atomic_get_mst_topology_state(state, mgr);
 
 	if (IS_ERR(mst_state))
-		return PTR_ERR(mst_state);
+		return -EINVAL;
 
 	list_for_each_entry(pos, &mst_state->vcpis, next) {
 
-- 
2.25.1

